src/app/core/session/session-service/local-session.ts

```diff
@@ -15,18 +15,12 @@
  *     along with ndb-core.  If not, see <http://www.gnu.org/licenses/>.
  */
 import { Injectable } from "@angular/core";
-import { LoginState } from "../session-states/login-state.enum";
-import {
-  encryptPassword,
-  LocalUser,
-  passwordEqualsEncrypted,
-} from "./local-user";
-import { SessionService } from "./session.service";
 import { PouchDatabase } from "../../database/pouch-database";
 import { AppSettings } from "../../app-settings";
 import { SessionType } from "../session-type";
 import { environment } from "../../../../environments/environment";
-import { AuthUser } from "./auth-user";
+import { AuthUser } from "../auth/auth-user";
+import { Database } from "../../database/database";
 
 /**
  * Responsibilities:
```

Comment:
the JsDoc might need an update.
Also, is "LocalSession" still the best name for this, as we are moving away from this core concept of a session, I think?

Reply:
Move to SessionManager